libs/designsystem/slide/src/slides.component.scss

```
@@ -20,6 +20,7 @@ $bullet-inactive-opacity: 0.2;
     justify-content: space-between;
     align-items: center;
     margin-block: utils.size('s');
+    margin-block-start: 0;
```

Consider shorthands:

```suggestion
    margin-block: 0 utils.size('s');
```

or simply

```suggestion
    margin: 0 utils.size('s') utils.size('s');
```

(letter latter requires you remove the next line (`margin-line` declaration)

---

Thanks for your suggestions! I personally prefer the first one